Rakefile

```diff
@@ -2,13 +2,15 @@ require "bundler/setup"
 require "bundler/gem_tasks"
 require "rake/testtask"
 require "bump/tasks"
-require "standard/rake"
 
-task lint: :standard
+desc "Format code"
+task :fmt do
+  sh "rubocop --auto-correct"
+end
 
 Rake::TestTask.new(:test) do |test|
   test.pattern = "test/*_test.rb"
   test.verbose = true
 end
 
-task default: [:test, :lint]
+task default: [:test, :fmt]
```

Comment:
I think we use :lint in other places, haven’t seen :fmt before 🤷🏼

Reply:
making it consistent between our go/ruby projects :D